examples/proc_nutation.py

```diff
@@ -0,0 +1,105 @@
+"""
+Process nutation data
```

Comment:
for this one, see the new `rough_table_of_integrals` function that I made.  The only thing that you need to do is to add the `fid_from_echo` function in the right place if the data is `echo_like`

Reply:
(that replaces most of the processing present in this file)